pyo3-macros-backend/src/pyfunction.rs

```diff
@@ -27,6 +27,11 @@ pub struct PyFunctionAttr {
     pub pass_module: bool,
 }
 
+#[derive(Clone, PartialEq, Debug)]
+pub struct PyFunctionArgAttrs {
+    pub from_py_with: Option<syn::ExprPath>,
```

Comment:
It looks like there's duplication here with `from_pyobject.rs`, both for this type declaration and the attribute parsing code itself.

Perhaps we can create a newtype in a new file `src/attrs.rs`, and put a newtype `struct FromPyWithAttribute(syn::ExprPath)` in there along with the parsing code which can be shared between the two?

Then here we can have:

```suggestion
    pub from_py_with: Option<FromPyWithAttribute>,
```

Reply:
Thanks for the suggestion! It's much cleaner this way